examples/simple_counter/iOS/project.yml

```diff
@@ -32,30 +32,5 @@ targets:
         UILaunchScreen: {}
     settings:
       OTHER_LDFLAGS: [-w]
-      SWIFT_OBJC_BRIDGING_HEADER: generated/sharedFFI.h
+      SWIFT_OBJC_BRIDGING_HEADER: ../shared/generated/swift/SharedFFI/sharedFFI.h
       ENABLE_USER_SCRIPT_SANDBOXING: NO
-    buildRules:
```

Comment:
BTW, I removed buildRules for "*.udl" since the "*.udl" file gone. It dependent on runining generate-swift first. I'm not familiar with xcodegen now. I'll investigate how to add a build phase.